mixer/pkg/adapter/test/integration.go

```diff
@@ -70,9 +70,9 @@ type (
 		GetState GetStateFn
 
 		// Templates supported by Mixer.
-		// If `tmpls` is not specified, the default templates inside istio.io/istio/mixer/template.SupportedTmplInfo
+		// If `Tmpls` is not specified, the default templates inside istio.io/istio/mixer/template.SupportedTmplInfo
 		// are made available to the Mixer.
-		tmpls map[string]template.Info
+		Tmpls map[string]template.Info
```

Comment:
As this is now exposed, would it be better named "Templates" for clarity?

Reply:
D